pom.xml

```diff
@@ -99,6 +99,14 @@
         <artifactId>maven-site-plugin</artifactId>
         <version>3.5</version>
       </plugin>
+      <plugin>
```

Comment:
Do we need this?

Reply:
This was added when moving from using the openjdk 9 to the oracle one (version 8). Not sure we need to keep this but without it the code didn't compile with `mvn`.